admin_solutions/authentication.adoc

```diff
@@ -5,10 +5,308 @@ toc::[]
+in the query portion of the URL. 
+.. Set the `*login*` parameter to *true* to redirect unauthenticated requests from clients expecting login flows. 
+.. Set the `*provider.loginURL*` parameter to the proxy URL to which to send clients expecting login flows. This parameter can include the 
+xref:../install_config/configuring_authentication.adoc#RequestHeaderIDP-urlquerytokens[`${url}` and `${query}` tokens]
+in the query portion of the URL. 
+.. Optionally, change the `*clientCA*` parameter to check incoming requests for a valid client certificate before the request's headers are checked for a user name. 
++
+[WARNING]
+====
+If you expect unauthenticated requests to reach the OAuth server, a `*clientCA*`
+parameter should be set for this identity provider. Otherwise, any direct
+request to the OAuth server can impersonate any identity from this provider,
+merely by setting a request header.
+====
+. Save your changes and close the file.
+. Start the {product-title} API server, specifying the configuration file you just modified:
++
+----
+$ openshift start master --config=<path/to/modified/config>/master-config.yaml
+----
+
+Once configured, any user logging in to the {product-title} web console will be
+prompted to log in using Request Header credentials.
+
+[[create-users-request-header-auth]]
+=== Creating Users with Request Header Authentication
+
+You do not create users in {product-title} when integrating with an external
+authentication provider, such as, in this case, Request Header. The Request
+Header authentication server is the system of record, meaning that users are
+defined there, and any user with a valid user name for the configured
+authentication server can log in.
+
+To add a user to {product-title}, the user must exist on the Request Header authentication server, and if required you must create a new Request Header authentication account for the user.
+
+[WARNING]
+====
+*[NEEDINFO]*: After googling "request header authentication", something tells me that the above section isn't true for this particular authentication method.
+====
+
+[[view-users-request-header-auth]]
+=== Viewing Users
+
+Once one or more users have logged in, you can run `oc get users` to view a
+list of users and verify that users were created successfully:
+
+.Output of `oc get users` command
+====
+
+----
+$ oc get users
+NAME         UID                                    FULL NAME   IDENTITIES
+bobsmith     a0c1d95c-1cb5-11e6-a04a-002186a28631   Bob Smith   requestheader:bobsmith <1>
+----
+<1> Identities in {product-title} are comprised of the identity provider name prefixed to the Request Header authentication username.
+====
+
+[WARNING]
+====
+*[NEEDINFO]*: Do you have access to a request header authentication setup? If so, can you please provide me with the output for the `oc get users` command when request header is configured as the auth provider for OpenShift? Thank you.
```

Comment:
the users that result from all the different auth methods are identical. they only vary based on the name that was chosen for the provider, and whether the provider makes display name info available.